build.sbt

```diff
@@ -63,21 +63,32 @@ lazy val `scala-native-jdbc-sqlite` = project
   .enablePlugins(ScalaNativePlugin)
   .dependsOn(`scala-native-jdbc`)
 
-lazy val `scala-native-jdbc-tests` = crossProject(JVMPlatform, NativePlatform)
+
+lazy val `scala-native-jdbc-duckdb` = project
+  .in(file("scala-native-jdbc-duckdb"))
+  .settings(
+    name := "scala-native-jdbc-duckdb"
+  )
+  .enablePlugins(ScalaNativePlugin)
+  .dependsOn(`scala-native-jdbc`)
+
+lazy val `scala-native-jdbc-sqlite-tests` = crossProject(JVMPlatform, NativePlatform)
```

Comment:
I changed the name of this project, since I needed the DuckDB tests to live in a separate project.

Reply:
Makes sense 👍